lib/Doctrine/DBAL/Driver/Mysqli/MysqliConnection.php

```diff
@@ -63,6 +63,19 @@ public function __construct(array $params, $username, $password, array $driverOp
 
         $this->_conn = mysqli_init();
 
+        if (isset($driverOptions['ssl_key']) &&
+            isset($driverOptions['ssl_cert']) &&
+            isset($driverOptions['ssl_ca'])
```

Comment:
Can you tell which of those 5 options are required at minimum to use this feature? I would like to have those evaluated and a `MysqliException` being thrown if one of the required options is missing, so that misconfiguration is exposed to the user. Unfortunately the PHP docs only state `Any unused SSL parameters may be given as NULL`. It doesn't give a word about which are definitely required.

Also we need test cases for all of the possibilities.